.gitignore

```diff
@@ -37,6 +36,8 @@ scipts/docker/nginx-conf/*
 build
 public/lib
```

Comment:
`public/lib` can be removed

Reply:
Yap, I put it back to be there for a little while so that simply pulling latest master wouldn't cause 1500 files to appear in `git status` when  `public/lib` disappears from gitignore.

Let's loop back to it and remove in a couple weeks.

Meanwhile, run `rm -fr public/lib` :-)